src/Public/Redis.ps1

```diff
@@ -62,16 +74,26 @@ Disconnect-Redis
 function Disconnect-Redis
 {
     [CmdletBinding()]
-    param()
+    param(
+        [Parameter()]
+        $ConnectionName
```

Comment:
Maybe as we spoke about, make this a `[string[]]` and foreach over them below?

Reply:
Done